portal/v2/src/ClusterDetailListComponents/MachinesList.tsx

```diff
@@ -199,11 +146,12 @@ export function MachinesListHelperComponent(props: {
     })
 
     setColumns(newColumns)
+    //setMachinesList(machineLocal)
```

Comment:
:( this means we had commented code on master, I guess for this reversion its fine and we plan on making changes to this again soon..